### PR TITLE
Laws rewrite. [DNM]

### PR DESCRIPTION
**Pull request by MistakeNot4892**
This will be a laws rewrite PR when I get around to splitting all the brain stuff out of it.

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 1]
```diff
@@ -31,92 +9,108 @@
...
+		to_chat(user, SPAN_WARNING("You upend \the [src], but the case is locked shut."))
+		return TRUE
 
-		SSstatistics.add_field("cyborg_mmis_filled",1)
+	if(!holding_brain)
+		to_chat(user, SPAN_WARNING("You upend \the [src], but there's nothing in it."))
+		return TRUE
 
-		return
+	to_chat(user, SPAN_NOTICE("You upend \the [src], spilling \the [holding_brain] onto \the [get_turf(src)]."))
```
Comment (quardbreak) [inline code/modules/mob/living/carbon/brain/MMI.dm]: `You open`

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 2]
```diff
@@ -31,92 +9,108 @@
...
-		brainobj = O
+	. = ..()
 
-		SetName("[initial(name)]: ([brainmob.real_name])")
-		update_icon()
+/obj/item/mmi/attack_self(mob/user)
 
-		locked = 1
+	if(locked)
+		to_chat(user, SPAN_WARNING("You upend \the [src], but the case is locked shut."))
```
Comment (quardbreak) [inline code/modules/mob/living/carbon/brain/MMI.dm]: `You open`

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 3]
```diff
@@ -31,92 +9,108 @@
...
+		return TRUE
 
-		SSstatistics.add_field("cyborg_mmis_filled",1)
+	if(!holding_brain)
+		to_chat(user, SPAN_WARNING("You upend \the [src], but there's nothing in it."))
```
Comment (quardbreak) [inline code/modules/mob/living/carbon/brain/MMI.dm]: `You open`

**Review by quardbreak — CHANGES_REQUESTED** [review]

**Comment by MistakeNot4892** [comment]
This has spun off into a refactor of laws as well, no idea when it'll be finished. God help me.

File: code/_onclick/hud/ai.dm [hunk 4]
```diff
@@ -76,13 +76,6 @@
 			/mob/living/silicon/ai/proc/ai_view_images
 			)
 
-	adding += new /obj/screen/ai_button(null,
```
Comment (MistakeNot4892) [inline code/_onclick/hud/ai.dm]: TODO readd

**Comment by MistakeNot4892** [comment]
Will reopen this when I get back to it.

**Pull request closed without merging**